101-azure-machine-learning-service-workspace/azuredeploy.json

```diff
@@ -0,0 +1,134 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "workspaceName": {
+            "type": "string",
+            "defaultValue": "myworkspace",
+            "metadata": {
+                "description": "The name of the Azure Machine Learning service workspace."
+            }
+        },
+        "workspaceLocation": {
+            "type": "string",
+            "defaultValue": "East US",
+            "allowedValues": [
+                "East US",
+                "East US 2",
+                "South Central US",
+                "Southeast Asia",
+                "West Central US",
+                "West Europe",
+                "West US 2"
+            ],
+            "metadata": {
+                "description": "The location where the workspace will be created."
+            }
+        }
+     },
+    "variables": {
+        "storageAccount": {
+            "name": "[concat('sa',uniqueString(resourceGroup().id))]",
+            "type": "Standard_LRS"
+        },
+        "keyVault": {
+            "name": "[concat('kv',uniqueString(resourceGroup().id))]",
+            "tenantId": "[subscription().tenantId]"
+        },
+        "applicationInsightsName": "[concat('ai',uniqueString(resourceGroup().id))]",
+        "containerRegistryName": "[concat('cr',uniqueString(resourceGroup().id))]"
+    },
+    "resources": [
+        {
+            "name": "[variables('storageAccount').name]",
+            "type": "Microsoft.Storage/storageAccounts",
+            "location": "[parameters('workspaceLocation')]",
+            "apiVersion": "2018-07-01",
+            "sku": {
+                "name": "[variables('storageAccount').type]"
+            },
+            "kind": "StorageV2",
+            "properties": {
+                "encryption": {
+                    "services": {
+                        "blob": {
+                            "enabled": true
+                        },
+                        "file": {
+                            "enabled": true
+                        }
+                    },
+                    "keySource": "Microsoft.Storage"
+                },
+                "supportHttpsTrafficOnly": true
+            }
+        },
+        {
+            "name": "[variables('keyVault').name]",
+            "type": "Microsoft.KeyVault/vaults",
+            "apiVersion": "2018-02-14",
+            "location": "[parameters('workspaceLocation')]",
+            "properties": {
+                "tenantId": "[variables('keyVault').tenantId]",
+                "sku": {
+                    "name": "standard",
+                    "family": "A"
+                },
+                "accessPolicies": [],
+                "enabledForTemplateDeployment": true,
```

Comment:
Are these flags are needed for this sample?  If not, remove them

Reply:
After checking, the accessPolicies is required, but the "enabled...." aren't. Will remove in next commit.